lib/pf/services/manager/dhcpd.pm

```diff
@@ -86,6 +87,31 @@ EOT
     return 1;
 }
 
+
+=head2 omapi_section
+
+Generate the omapi section if it is defined
+
+=cut
+
+sub omapi_section {
+    return '' if isdisabled($Config{advanced}{use_omapi_to_lookup_mac});
```

Comment:
It will help diagnose if something is not right.
We will not start to argue on the purpose of comments, I think you already know my position ;)